### PR TITLE
docs: update release-notes.adoc for launch

### PR DESCRIPTION
**Pull request by marcobonita**

### CODE REVIEW

**Review by AurePellefigue — COMMENTED** [review]
it looks good to me

File: modules/ROOT/pages/release-notes.adoc [hunk 1]
```diff
@@ -11,12 +11,28 @@ The {bonitaVersion} version is in development.
...
 
+We’re proud to announce Bonita {bonitaVersion}, the first Bonita platform release of this year! 
+This new major release improves security, search capabilities and performance.
+
+== The {bonitaVersion} release in a nutshell
+
+* Java 17 update for more security, better performance and language enhancements
+* Improved security in Bonita REST APIs
+* Boosted search experience on all APIs and pages
+* Better platform performance thanks to the L2 Cache (Enterprise Edition only)
```
Comment (rbioteau) [inline modules/ROOT/pages/release-notes.adoc]: > Better platform performance thanks to the L2 Cache (Enterprise Edition only)

@educhastenier  Isn't the improvement about clustered L2 Cache ?

File: modules/ROOT/pages/release-notes.adoc [hunk 2]
```diff
@@ -11,12 +11,28 @@ The {bonitaVersion} version is in development.
 The required Java version is now **Java 17**.
 ====
 
+We’re proud to announce Bonita {bonitaVersion}, the first Bonita platform release of this year!
```
Comment (educhastenier) [inline modules/ROOT/pages/release-notes.adoc]:
```suggestion
We’re proud to announce Bonita {bonitaVersion}, the first major Bonita platform release of this year! 
```

File: modules/ROOT/pages/release-notes.adoc (hunk 1 above)
Comment (educhastenier) [inline modules/ROOT/pages/release-notes.adoc]:
```suggestion
* Better platform performance thanks to the L2 Cache (Enterprise Edition only), extended to clustered environments.
```

File: modules/ROOT/pages/release-notes.adoc [hunk 3]
```diff
@@ -46,10 +62,12 @@ For more information on how to tune the L2 Cache with Bonita, please refer to th
 
 ==== Like-based search
 
-Bonita Runtime search capabilities in APIs and application pages have been improved by changing the algorithm to support "like-based" search. +
-With the "like-based" algorithm you can find all the matching records for which the search string occurs anywhere in a phrase or a word. +
+Bonita {bonitaVersion} introduces a new search engine algorithm on all APIs and the Bonita pages included by default on Bonita User Application, Bonita Administrator and Super Administrator Applications. This new “like-based” search algorithm replaces the previous “word-based” search and it will allow you to rapidly find all matching records for a given search string anywhere in a phrase or a word.
```
Comment (educhastenier) [inline modules/ROOT/pages/release-notes.adoc:65]: Add a link to the detailling page

**Review by educhastenier — CHANGES_REQUESTED** [review]
minor change requested

File: modules/ROOT/pages/release-notes.adoc (hunk 3 above)
Comment (educhastenier) [reply modules/ROOT/pages/release-notes.adoc:65]: @marcobonita Why did you resolve this comment without any change or remark?

**Review by educhastenier — APPROVED** [review]

**Pull request merged**